justfile

```diff
@@ -190,6 +190,11 @@ ci-musl-prepare: ci-prepare
     exit 1
 
 version-up version:
+    #!/usr/bin/bash
+    set -ex
+    if [[ "{{version}}" == "0.3.0" ]]; then
+        return
+    fi
```

Comment:
Why do we need it?
```suggestion
```

Reply:
The main thing is that when this PR is submitted, a current version of the PR will be automatically generated. Another way is to change the current default version number to 0.3.1

Reply:
It's a good time to release 0.3.1

Reply:
Of course, I've deleted it. When the current PR is merged, it will automatically create a new PR. When the new PR is merged, it can be automatically released.